epidose/back_end/deploy.yml

```diff
@@ -95,3 +95,16 @@
       systemd:
         name: nginx
         state: restarted
+
+    - name: obtain certificate
+      become: yes
+      become_user: root
+      shell: certbot --nginx --non-interactive -d {{ server_name }} --agree-tos --email stefanos1316@gmail.com --email dds@aueb.gr
+
+    - name: add cron entry to renew cert every 12 hours
+      become: yes
+      become_user: root
+      cron:
+        name: update cert every 12 hours
```

Comment:
Every 4 days is more than enough

Reply:
Christos from Eellak told me to invoke a renewal request every 12 hours,
but I will change it to four days as you said.

Reply:
Have changed to run every four days and this is the `crontab -e` output that I get:
`* * */4 * * /usr/bin/certbot renew --quiet && service nginx reload`
Is this okay?